src/appleseed/renderer/modeling/camera/camera.cpp

```diff
@@ -380,6 +423,38 @@ DictionaryArray CameraFactory::get_input_metadata()
             .insert("use", "optional")
             .insert("default", "0.0"));
 
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "shutter_open_end_time")
+            .insert("label", "Shutter Open End Time")
+            .insert("type", "numeric")
+            .insert("min",
+                Dictionary()
+                    .insert("value", "0.0")
+                    .insert("type", "soft"))
+            .insert("max",
+                Dictionary()
+                    .insert("value", "1.0")
+                    .insert("type", "soft"))
+            .insert("use", "optional")
+            .insert("default", "0.2"));
```

Comment:
The default values must be such that the existing behavior is preserved (infinitely fast opening/closing of the shutter).